Make note of 16 probe limit

NHodgesVFX opened the pull request:
Godot by default has a limit of 16 reflection probes in a scene. this can be increased by increasing atlas subdiv. For 3.2 and possibly 4.0. This also fixes a spelling mistake

clayjohn commented on doc/classes/ReflectionProbe.xml:
```diff
@@ -4,8 +4,9 @@
-		Capture its surroundings as a dual parabolid image, and stores versions of it with increasing levels of blur to simulate different material roughnesses.
+		Capture its surroundings as a dual paraboloid image, and stores versions of it with increasing levels of blur to simulate different material roughnesses.
 		The [ReflectionProbe] is used to create high-quality reflections at the cost of performance. It can be combined with [GIProbe]s and Screen Space Reflections to achieve high quality reflections. [ReflectionProbe]s render all objects within their [member cull_mask], so updating them can be quite expensive. It is best to update them once with the important static objects and then leave them.
+		Note: By default Godot will only render 16 reflection probes. If you need more increase Atlas Subdivs, This setting can be found in Project Settings -> Quality -> Reflections 
```
You can actually link directly to project settings using the suggested syntax below. :)

```suggestion
		Note: By default Godot will only render 16 reflection probes. If you need more, increase the number of atlas subdivisions. This setting can be found in [member ProjectSettings.rendering/quality/reflections/atlas_subdiv].
```

NHodgesVFX commented:
Thanks @clayjohn I will make sure to do that in the future.

clayjohn reviewed (approved):
Woohoo! First PR to the main repo.

akien-mga commented:
Could you squash commits into one?

Also, PRs should normally be done against the master branch, not 3.2. I cherrypick the relevant changes to stable branches once merged in master.

Not sure if the Vulkan port still had the same restriction though, WDYT @clayjohn?

NHodgesVFX commented:
ill squash it, also @clayjohn mentioned that it should be for 3.2 here in a similar pull https://github.com/godotengine/godot-docs/pull/3345#issuecomment-609111959
Not sure if thats because I put 3.2 in the title or if it doesnt apply.

NHodgesVFX committed:
Make note of 16 probe limit

Godot by default has a limit of 16 reflection probes in a scene. this can be increased by increasing atlas subdiv. For 3.2 and possibly 4.0. This also fixes a spelling mistake

Update doc/classes/ReflectionProbe.xml

Fix from clayjohn, Directly link to settings

Co-Authored-By: Clay John <claynjohn@gmail.com>

akien-mga commented:
Thanks!
